### DISCUSSION TITLE
MissionControl 0.11.1

### DISCUSSION THREAD

**Discussion by ndeadly** [ndeadly/MissionControl#826] [General]
<p align="left">
<img alt="GitHub Releases" src="https://img.shields.io/github/downloads/ndeadly/MissionControl/v0.11.1/total">
<a href="https://discord.gg/gegfNZ5Ucz"><img alt="Discord Server" src="https://discordapp.com/api/guilds/905069757361971303/widget.png?style=shield"/></a>
</p>

# ❗****Important: This release requires Atmosphere version 1.7.0 or later****❗

This is the twenty-fourth official release of Mission Control.

This is just a quick release to add support for firmware 18.1.0. No new features this time around (plenty in the works, though!).

New users landing on this release page should first check out the readme on the [main project page](https://github.com/ndeadly/MissionControl#readme) for the official project documentation. There you can find installation and usage instructions along with an FAQ section that will answer most of your questions.

_PS: there is some ongoing testing of preliminary Bluetooth LE  and USB support happening in my discord server (see #testing-ble and #testing-usb channels). If you own any of the controllers listed [here](https://github.com/ndeadly/MissionControl/issues?q=is%3Aissue+is%3Aopen+label%3A%22BLE+controller%22) or any BLE or USB controllers that are not on the list, we'd like to hear from you! (though I don't need any additional input on Xbox and Stadia controllers for now). Please note that these are alpha builds for the purpose of gathering controller information and furthering my understanding of how BLE/USB works on the console. They are **_NOT_** intended for general use. There _will_ be bugs and issues. There are no ETAs. I will not be providing any support - if you can't follow the usage instructions, then this is not for you._

### Changelog:
- Added support for firmware 18.1.0.

<hr /><em>This discussion was created from the release <a href='https://github.com/ndeadly/MissionControl/releases/tag/v0.11.1'>MissionControl 0.11.1</a>.</em>

**Comment by beidouxiaonan** [comment]
the version of my switch  is 18.1.0, but there is still no response when connecting the PS5 controller.

**Reply by ndeadly** [reply]
Then there is a problem with your installation. Try the archive bit fix in Hekate

**Reply by beidouxiaonan** [reply]
oh my god!  i didn't expect you to be online. Could you be more specific?

**Reply by beidouxiaonan** [reply]
Master, could you make a tutorial?

**Reply by ndeadly** [reply]
In Hekate under the Tools menu there is an option to fix the archive bits on your SD card 
![image](https://github.com/ndeadly/MissionControl/assets/24677491/7e67972c-f63f-4861-9892-7eb04b1180c9)

**Reply by beidouxiaonan** [reply]
<img width="1073" alt="image" src="https://github.com/ndeadly/MissionControl/assets/18432261/12af9982-e954-405e-8c9a-4622d0aad5df">
master，my hekate have no this option,is somewhere wrong?

**Reply by ndeadly** [reply]
I can't read Chinese, but I think you need to tap the first text option next to where it says RCM in the bottom right of the screen, from memory.

**Reply by beidouxiaonan** [reply]
sorry about that, I have found it according to the icon you circled。THX！you are super cool!

**Reply by philip-pc** [reply]
> In Hekate under the Tools menu there is an option to fix the archive bits on your SD card ![image](https://private-user-images.githubusercontent.com/24677491/344111506-7e67972c-f63f-4861-9892-7eb04b1180c9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM0MDc4NjYsIm5iZiI6MTcyMzQwNzU2NiwicGF0aCI6Ii8yNDY3NzQ5MS8zNDQxMTE1MDYtN2U2Nzk3MmMtZjYzZi00ODYxLTk4OTItN2ViMDRiMTE4MGM5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODExVDIwMTkyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBlNTM5NWU1ZTdiMTNkMTdjNzhmZjc2YTkwYTI3MGRiMjFlMzE0ZWNkZDYxNTY5YzlkNzBmMGVmOWNiZjZmOTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2GMyohYN_jP-0gkQgAe2nXkJADTxd9alBmyZXM5N5rM)

Instant fixed! I've spend 6hrs reinstalling AMS in different versons...should've seen this article sooner.

**Reply by Eddieus16** [reply]
大哥 恢复eMMC就是> <img width="1073" alt="image" src="https://github.com/ndeadly/MissionControl/assets/18432261/12af9982-e954-405e-8c9a-4622d0aad5df">
> 
> master，my hekate have no this option,is somewhere wrong?

**Reply by ndeadly** [reply]
It should be under one of those options in the bottom right where it says RCM

**Comment by beidouxiaonan** [comment]
您好，亲的邮件已经收到。我会尽快处理。

**Comment by S1-SLASH1** [comment]
for dual sense vibration is week. is it possible to increase it?

**Reply by ndeadly** [reply]
Yes, you can change [this setting](https://github.com/ndeadly/MissionControl/blob/581ae0d9e89018fc0827dcd170082b6a3bd0263c/mc_mitm/config.ini#L28) in the module configuration .ini file.

See https://github.com/ndeadly/MissionControl?tab=readme-ov-file#module-configuration for more details about the configuration in general

**Comment by ActiveSierro** [comment]
quick question ive updated everything to the latest version and i play smash ultimate with my friends but we use ps4 controllers to play and its flawless but when we disconnect the player using css meaning the character select screen it will crash the whole game any reason why ?

**Reply by ndeadly** [reply]
Not sure, I would need to see an error report to know where it's originating from

**Reply by ActiveSierro** [reply]
how do i check ?

**Reply by ndeadly** [reply]
Depends on the nature of the error. Does it crash the console and throw up an error screen? Or just exits the game with a generic "error occurred" message? I think all types generate reports in subdirectories under /atmosphere on your SD card, but they're in different places depending on which type

**Comment by Seikashii** [comment]
When I launch the console, the Mission Control software crashes the switch. How can I fix this?

**Reply by ndeadly** [reply]
You most likely either need to update mission control, or atmosphere (or both)

**Reply by Seikashii** [reply]
Im in 17.0.0, and I download this version of Mission Control

**Reply by ndeadly** [reply]
I'd wager you're not using the latest atmosphere then

**Reply by Seikashii** [reply]
U think if I download the latest, mission control work ? U wright "requires 17.0.0 or later" so Im good no ?

**Reply by ndeadly** [reply]
17.0.0 is a console firmware version. Atmosphere 1.7.0 is what I wrote. I'm fairly confident you are using an outdated Atmosphere

**Reply by Seikashii** [reply]
Where I can find the version of my Atmosphere pls ?

**Reply by ndeadly** [reply]
It's displayed in the system menu underneath where you would trigger a system update

**Reply by Seikashii** [reply]
U right Im in 1.6.1, Im gonna try to update

**Comment by coltsito** [comment]
I have a problem, the hekate software stopped with an error "desc std abort 0xffe and program id 0100000000bd0", even though I followed the tutorials as indicated by its installation and I had the latest version, that came up, what I did was search for that error by hand and eliminate it, it worked although now my joycons don't connect to the console remotely, my technician who installed everything for me never sent me a backup and I can only play portablely, I hope you can help me, thanks.

**Comment by PalLaDiNeee** [comment]
I would like to ask when support for xbox controller series x/s will be available

**Reply by ndeadly** [reply]
When it's complete. There are already test builds available on my discord server.

**Comment by beidouxiaonan** [comment]
您好，亲的邮件已经收到。我会尽快处理。

**Comment by aratros** [comment]
Hi guys, recently I have bought a 8Bitdo NEOGEO Wireless controller, it works perfectly under windows and android, but I have tried to sincronize it with my switch using mission control at it doesn't work, it has 2 modes, 2.4 gz and bluetooth but it doesn't work, any tip, or maybe is just that is not supported in the current version of the software?

**Reply by ndeadly** [reply]
It's a Bluetooth LE controller and won't work with the current release versions. There are experimental builds with BLE support on my discord server, and the neo geo controller should be working

**Reply by aratros** [reply]
Could I have an invitation to your server?, I would like to give it a try.

**Reply by Banz99** [reply]
> Could I have an invitation to your server?, I would like to give it a try.

https://discord.gg/gegfNZ5Ucz

**Comment by akralail** [comment]
i can't connect my Xbox controller series s ??

**Reply by ndeadly** [reply]
These use Bluetooth LE, which isn't compatible with the release builds

**Comment by beidouxiaonan** [comment]
您好，亲的邮件已经收到。我会尽快处理。

**Comment by khalilou2026** [comment]
Is MissionControl 0.11.1 supports firmware 19.0.0 ?

**Reply by ndeadly** [reply]
No, it doesn't. I'm currently waiting for atmosphere libraries to be updated so I can make a new release

**Reply by Ch3ck3rM0n** [reply]
isnt it integrated in the pre-preversion-release, yet?

**Reply by ndeadly** [reply]
Atmosphere releases are independent to library updates. The latter still hasn't been committed

**Reply by Ch3ck3rM0n** [reply]
ah thx, didnt know that. i am not so deep in the programming topic. so i will remove mission controll till you get the new version, take your time :)
